hspec-golden.cabal

```diff
@@ -37,6 +37,7 @@ library
   build-depends:
       base >=4.6 && <5
     , directory
+    , filepath >=1.0 && <2.0
```

Comment:
Could you please add this dependency to the [package.yml](https://github.com/stackbuilders/hspec-golden/blob/master/package.yaml#L24) file? The cabal file is auto-generated.